src/contact/dto/create-contact.dto.ts

```diff
@@ -0,0 +1,8 @@
+import { IsString } from 'class-validator';
+
+export class CreateContactDto {
+  @IsString()
+  name: string;
```

Comment:
Add blank line after each property definition

Reply:
the blank lines were added